addons/l10n_ph/migrations/1.2/end-migrate_update_taxes.py

```diff
@@ -0,0 +1,34 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, SUPERUSER_ID
+
+TAX_DESCRIPTION_MAP = {
+    "WV080": "Final Withholding VAT on Purchase of Digital Service consumed in the Philippines from the non-resident digital service providers (Private Withholding Agent)",
+    "WV100": "Final Withholding VAT on the gross amount by resident e-marketplace to the non-resident sellers/merchants for the digital services sold/paid through their platform/facility",
+    "WV010": "VAT Withheld by the government or any of its political subdivisions, instrumentalities, or agencies including GOCCs",
+    "WV020": "VAT Withheld by the government or any of its political subdivisions, instrumentalities, or agencies including GOCCs",
+    "WI011": "Professional Fees",
+    "WI100": "Gross rental of property",
+    "WI120": "Contractors",
+    "WI516": "Commission / Rebates / Discounts",
+    "WC011": "Professional Fees",
+    "WC100": "Gross rental of property",
+    "WC516": "Commission / Rebates / Discounts",
+    "WC820": "Gross remittances by e-marketplace operators to the sellers/merchants for the goods or services sold/paid through their platform/facility",
+    "WC830": "Gross remittances by digital financial services providers to the sellers/merchants for the goods or services sold/paid through their platform/facility",
+    "WC230": "Other payments to Non-Resident Foreign Corporations (NRFCs)",
+}
+
+
+def migrate(cr, version):
+    # Remove ATC codes from the beginning of tax descriptions in l10n_ph.
+    for atc_code, desc in TAX_DESCRIPTION_MAP.items():
+        cr.execute("""
+            UPDATE account_tax
+            SET
+                description = jsonb_build_object('en_US', %s)
+            WHERE l10n_ph_atc = %s
+        """, (desc, atc_code))
+
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    for company in env['res.company'].search([('chart_template', '=', 'ph')], order="parent_path"):
+        env['account.chart.template'].try_loading('ph', company)
```

Comment:
We should never do this in a module update, this is too broad and can change user data without the user realizing it.